react/components/LogoutCustomerSession.tsx

```diff
@@ -49,19 +53,46 @@ interface Props {
 const LogoutCustomerSession = (props: Props) => {
   const { client, loading, onDepersonify, attendantEmail, mobile } = props
   const handles = useCssHandles(CSS_HANDLES)
+  const { orderForm } = useOrderForm()
 
-  const getClientName = (client: any) =>
-    !!client
-      ? client.name.includes('null')
-        ? client.email.slice(0, client.email.indexOf('@'))
-        : client.name
-      : null
+  const { id } = orderForm
 
-  const getHeader = (mobile: boolean) => {
+  const [customerClass, setCustomerClass] = useState('')
+
+  useEffect(()=> {
+    async function getCustomerClass() {
+      const requestOptions = {
+        method: 'GET',
+      }
+      
+      const data = await fetch(`/api/checkout/pub/orderForm/${id}`, requestOptions).then(res => res.json())
+      
+      if (!!data?.clientProfileData?.customerClass) {
+        setCustomerClass(data.clientProfileData.customerClass)
+      }
+      
+    }
+
+    getCustomerClass()
+  }, [])
+
+  const getClientName = (user: any) => {
+    if (!!user) {
+      if (user.name.includes('null')) {
+        return user.email.slice(0, user.email.indexOf('@'))
+      } else {
+        return user.name
+      }
+    } else {
+      return null
+    }
```

Comment:
```suggestion
    if (!user) {
       return null 
    }
    
    if (user.name.includes('null')) {
        return user.email.slice(0, user.email.indexOf('@'))
    }
    
    return user.name
```

This code seems a little bit easier to read, wdyt about it?

Reply:
Yes, this seems easier to read. I have updated the code.

Reply:
@igorbrasileiro Yes, this seems easier to read. I have updated the code.